src/code/audio_load.c

```diff
@@ -790,83 +791,145 @@ AudioTable* AudioLoad_GetLoadTable(s32 tableType) {
     return table;
 }
 
-void AudioLoad_RelocateFont(s32 fontId, SoundFontData* mem, RelocInfo* relocInfo) {
-    u32 reloc;
-    u32 reloc2;
+/**
+ * Read and extract information from soundFont binary loaded into ram.
+ * Also relocate offsets into pointers within this loaded soundFont
+ *
+ * @param fontId index of font being processed
+ * @param fontData ram address of raw soundfont binary loaded into cache
+ * @param relocInfo information on the sampleBank containing raw audio samples
+ */
+void AudioLoad_RelocateFont(s32 fontId, SoundFontData* fontData, SampleBankRelocInfo* relocInfo) {
+    u32 soundOffset;     // Relative offset from the beginning of fontData directly to the sound/envelope
+    u32 soundListOffset; // Relative offset from the beginning of fontData to the list of soundOffsets/sfxs
     Instrument* inst;
     Drum* drum;
     SoundFontSound* sfx;
     s32 i;
     s32 numDrums = gAudioContext.soundFonts[fontId].numDrums;
     s32 numInstruments = gAudioContext.soundFonts[fontId].numInstruments;
     s32 numSfx = gAudioContext.soundFonts[fontId].numSfx;
-    void** ptrs = (void**)mem;
+    void** fontDataStartAddr = (void**)fontData;
```

Comment:
I think I would swap these names around,
```suggestion
    void** fontData = (void**)fontDataStartAddr;
```
as the more opaque type (purely an address) is `fontDataStartAddr`,
and `fontData` is actually used to read/write pointers

Also `fontData` is used as the reloc target so `fontDataStartAddr` would work great for it